AirBench/Scripts/Home.js

```diff
@@ -16,10 +16,24 @@
         return await fetch("/api/bench/all");
     }
 
+    function clickMarker(event){
+        //console.log('click');
+        //console.log(event.pixel);
+        //console.log(map.getCoordinateFromPixel(event.pixel));
+        var lonlat = ol.proj.transform(event.coordinate, 'EPSG:3857', 'EPSG:4326');
+        console.log(lonlat[0]);
+        document.cookie = "lat=" + lonlat[1] + ";path=/";
+        document.cookie = "lon=" + lonlat[0] + ";path=/";
+        console.log(document.cookie);
+        document.location.href = "/Bench/Add";
```

Comment:
Nice job coming up with a solution for passing the lat/lon to the next page. Instead of using a cookie, why not just pass the lat/lon as query string parameters to the `/Bench/Add` path (i.e. `/Bench/Add?Lat=###&Lon=###`? Your action method can add nullable floating point number type `lat` and `lon` parameters and pass those to the view when they're set.